packages/webdriver/protocol/chromium.json

```diff
@@ -414,5 +414,36 @@
         "description": "The base64-encoded PNG image data comprising the screenshot of the visible region of an element’s bounding rectangle after it has been scrolled into view."
       }
     }
+  },
+  "/session/:sessionId/se/log/types": {
+    "GET": {
+      "command": "getLogTypes",
+      "description": "Get available log types.",
+      "ref": "https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidlogtypes",
+      "parameters": [],
+      "returns": {
+        "type": "String[]",
+        "name": "logTypes",
+        "description": "The list of available log types."
```

Comment:
Let's add the few log types as example in here

Reply:
done